traffic_ops/traffic_ops_golang/deliveryservice/servers/servers.go

```diff
@@ -97,130 +100,107 @@ func (dss *TODeliveryServiceServer) Validate(db *sqlx.DB) []error {
 	return tovalidate.ToErrors(errs)
 }
 
-//The TODeliveryServiceServer implementation of the Creator interface
-//all implementations of Creator should use transactions and return the proper errorType
-//ParsePQUniqueConstraintError is used to determine if a profileparameter with conflicting values exists
-//if so, it will return an errorType of DataConflict and the type should be appended to the
-//generic error message returned
-//The insert sql returns the profile and lastUpdated values of the newly inserted profileparameter and have
-//to be added to the struct
-func (dss *TODeliveryServiceServer) Create(db *sqlx.DB, user auth.CurrentUser) (error, tc.ApiErrorType) {
-	rollbackTransaction := true
-	tx, err := db.Beginx()
-	defer func() {
-		if tx == nil || !rollbackTransaction {
-			return
+// api/1.1/deliveryserviceserver$
```

Comment:
This breaks GoDoc parsing, a comment above a function needs to start with the name of the function, i.e. `// ReadDSSHandler is served at api/1.1/deliveryserviceserver`. Alternatively, you can put the comment somewhere else, like inside the function, to avoid breaking GoDoc.